api/src/org/apache/cloudstack/api/command/user/vpn/AddVpnUserCmd.java

```diff
@@ -124,7 +124,7 @@ public void execute() {
                 throw new ServerApiException(ApiErrorCode.INTERNAL_ERROR, "Failed to add vpn user");
             }
         }catch (Exception ex) {
```

Comment:
I still do not understand the reason for this catch/all here.
Could you elaborate a bit more?